torch/csrc/jit/script/compiler.cpp

```diff
@@ -2241,8 +2241,39 @@ struct to_ir {
         auto values = getValues(ll.inputs(), /*maybe_unpack=*/true);
         return graph->insertNode(graph->createTuple(values))->output();
       } break;
+      case TK_DICT_LITERAL: {
+        auto dl = DictLiteral(tree);
+        auto keys = getValues(dl.key_inputs(), /*maybe_unpack=*/true);
```

Comment:
Evaluation order warning: if I write
`{ print('a'): print('b'), print('c'), print('d') }`. Then it should print `a b c d` but this will print `a c b d`.